<?xml version='1.0' encoding='UTF-8'?>
<rss xmlns:atom="http://www.w3.org/2005/Atom" xmlns:content="http://purl.org/rss/1.0/modules/content/" version="2.0">
  <channel>
    <title>CVE Feed for ansys -- all</title>
    <link>https://raw.githubusercontent.com/deepseas/cvelistV5monitor/main/feeds/ansys/all.rss</link>
    <description>The latest CVEs for ansys -- all products</description>
    <docs>http://www.rssboard.org/rss-specification</docs>
    <generator>python-feedgen</generator>
    <lastBuildDate>Tue, 04 Jun 2024 17:29:29 +0000</lastBuildDate>
    <ttl>60</ttl>
    <item>
      <title>CVE-2024-29189|2024-03-26T02:50:34.984Z -- ansys -- pyansys-geometry
</title>
      <link>https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2024-29189</link>
      <description>PyAnsys Geometry is a Python client library for the Ansys Geometry service and other CAD Ansys products. On file src/ansys/geometry/core/connection/product_instance.py, upon calling this method _start_program directly, users could exploit its usage to perform malicious operations on the current machine where the script is ran. This vulnerability is fixed in 0.3.3 and 0.4.12.</description>
      <guid isPermaLink="false">CVE-2024-29189|2024-03-26T02:50:34.984Z</guid>
      <pubDate>Tue, 26 Mar 2024 02:50:34 +0000</pubDate>
    </item>
  </channel>
</rss>
